view/theme/frio/templates/admin/contactblock.tpl

```diff
@@ -0,0 +1,63 @@
+
+<script>
```

Comment:
can you please insert the js part in a separate file `view/theme/frio/js/mod_admin.js`?
For frio I want to avoid (if it's possible) to have js code in the templates. A separate file would be cleaner and better to debug and we could have a better overview of the used js code.

Reply:
Gotcha, will do.